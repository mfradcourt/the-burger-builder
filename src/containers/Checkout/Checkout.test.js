import React from 'react';
import { shallow } from "enzyme";
import { createMemoryHistory, createLocation } from 'history';

import Checkout from './Checkout';

describe('<Checkout />', () => {
  let wrapper;

  it('should render Checkout', () => {
    const history = createMemoryHistory();
    const path = `/checkout`;

    const match = {
      isExact: false,
      path,
      url: path,
      params: { price: 4.00, ingredients: {} }
    };

    const location = createLocation(match.url);

    wrapper = shallow(
      <Checkout
        location={location}
        match={match}
        history={history}
      />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
