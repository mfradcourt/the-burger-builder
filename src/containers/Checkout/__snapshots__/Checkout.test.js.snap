// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Checkout /> should render Checkout 1`] = `
<div>
  <checkoutSummary
    checkoutCancelled={[Function]}
    checkoutContinued={[Function]}
    ingredients={Object {}}
  />
  <Route
    path="/checkout/contact-data"
    render={[Function]}
  />
</div>
`;
